okhttp-mockwebserver/src/main/kotlin/okhttp3/mockwebserver/internal/duplex/MockWebServerAsserts.kt

```diff
@@ -0,0 +1,31 @@
+package okhttp3.mockwebserver.internal.duplex
+
+open class MockWebServerAsserts {
```

Comment:
@swankjesse McDonald's suggestion for avoiding any dependencies inside of MockWebServer module